media/css/sandstone/sandstone.css

```diff
@@ -558,186 +558,186 @@ nav.menu-bar span:first-child a {
  * Tables without this class don't have any special styling.
  */
 .table {
-    border-collapse: collapse;
-    border-spacing: 0;
-    margin-bottom: 20px;
+  border-collapse: collapse;
+  border-spacing: 0;
+  margin-bottom: 20px;
 }
 
 .table caption {
-    text-align: left;
-    padding-bottom: 1px; /* fix to get it to look nice in many browsers */
-    margin-bottom: -1px;
-    width: 100%;
+  margin-bottom: -1px;
+  padding-bottom: 1px;  /* Fix to get it to look nice in many browsers. */
+  text-align: left;
+  width: 100%;
 }
 
 .table th,
 .table td {
-    border-top: 1px solid rgba(0,0,0,0.2);
-    padding: 10px 20px 10px 0;
-    text-align: left;
+  border-top: 1px solid rgba(0,0,0,.2);
+  padding: 10px 20px 10px 0;
+  text-align: left;
 }
 
 .table thead th,
 .table thead td {
-    border-top: 0;
-    padding-top: 0;
-    font: normal 16px 'Open Sans Light', sans-serif;
+  border-top: 0;
+  font: normal 16px 'Open Sans Light', sans-serif;
+  padding-top: 0;
 }
 
 
 /* Sidebar */
 .sidebar nav {
-    font: normal 16px 'Open Sans Light', sans-serif;
-    color: #484848;
+  color: #484848;
+  font: normal 16px 'Open Sans Light', sans-serif;
 }
 
 .sidebar nav li {
-    list-style-type: none;
-    border-bottom: 1px dotted #ccc;
-    margin: 0;
+  border-bottom: 1px dotted #ccc;
+  list-style-type: none;
+  margin: 0;
 }
 
 .sidebar nav a,
 .sidebar nav b {
-    display: block;
-    padding: 8px 0;
+  display: block;
+  padding: 8px 0;
 }
 
 .sidebar nav li b {
-    font-weight: bold;
+  font-weight: bold;
 }
 
 
 /* Footer */
 #colophon {
-    color: #666;
-    padding: 40px 0;
-    font-size: .875em;
-    line-height: 1.285;
-    background: #fff;
-    width: 100%;
-    margin: 1em 0 0;
+  background: #fff;
+  color: #666;
+  font-size: .875em;
+  line-height: 1.285;
+  margin: 1em 0 0;
+  padding: 40px 0;
+  width: 100%;
 }
 
 #colophon .row {
-    width: 960px;
-    margin: 0 auto;
+  margin: 0 auto;
+  width: 960px;
 }
 
 #colophon .row:after {
-    content: "";
-    display: block;
-    clear: both;
-    height: 0;
-    visibility: hidden;
+  clear: both;
+  content: "";
+  display: block;
+  height: 0;
+  visibility: hidden;
 }
 
 #colophon .footer-logo,
 #colophon .footer-license,
 #colophon .footer-nav {
-    float: left;
-    margin: 0 10px;
+  float: left;
+  margin: 0 10px;
 }
 
 #colophon h1.logo {
-    font-size: .875rem;
-    margin-bottom: 10px;
+  font-size: .875rem;
+  margin-bottom: 10px;
 }
 
 .footer-license-text {
-    color: #999;
+  color: #999;
 }
 
 #colophon .footer-logo {
-    width: 300px;
+  width: 300px;
 }
 
 #colophon .footer-license {
-    width: 220px;
+  width: 220px;
 }
 
 #colophon .footer-nav {
-    width: 140px;
+  width: 140px;
 }
 
 #colophon a,
 #colophon a:link,
 #colophon a:visited {
-    color: #0096dd;
+  color: #0096dd;
```

Comment:
this seems to be the exact same colour as used for the default `a:link, a:visited` styles. can remove. (and same for next block.)